common/types.go

```diff
@@ -100,3 +102,110 @@ const (
 	// in a request context.
 	RequestIDContextKey ContextKey = "request_id"
 )
+
+var (
+	// ErrNetworkUnknown is returned if a chain context does not correspond
+	// to a known network identifier.
+	ErrNetworkUnknown = errors.New("network unknown")
+
+	// ErrRuntimeUnknown is returned if a chain context does not correspond
+	// to a known runtime identifier.
+	ErrRuntimeUnknown = errors.New("runtime unknown")
+)
+
+// ChainID is the ID of a chain.
+type ChainID string
+
+// String returns the string representation of a ChainID.
+func (c ChainID) String() string {
+	return string(c)
+}
+
+// Network is an instance of the Oasis Network.
+type Network uint8
```

Comment:
currently unused. let's replace with a new `type Network string` like how `Runtime` is and start using it in places where we use `"mainnet"` etc. literals

Reply:
![image](https://user-images.githubusercontent.com/38229038/233752365-578df8a8-8eba-4935-9a1c-40d56542d3c3.png)

yeah it's used a couple of times as "chain name" in (1) the definition of the default chains and (2) in tests

Reply:
#387